_posts/2022-09-10-executive-director-pyopensci-leah-wasser-updates.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: single
+title: "Big Changes for pyOpenSci - an update from the new executive director, leah wasser, what's next?"
+excerpt: "pyOpenSci has recently transitioned to a fiscal sponsor and has a new executive director - Leah Wasser. Learn more about our goals over the next few years and how to get involved with the community."
+author: "Leah Wasser"
+permalink: /blog/new-executive-director-leah-wasser
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - updates
+  - blog-post
+---
+
+
+## pyOpenSci is now an independent community organization
+
+I am thrilled to announce that pyOpenSci is now a fiscally sponsored project of 
+Community Initiatives! This means that pyOpenSci now serves the community of 
+scientific python users and open source software maintainers. pyOpenSci is not 
+affiliated with any specific organization. I (Leah) will now be devoting ALL of my 
+(work :)) time to the pyOpenSci project full time as the official Executive Director. 
+
+## pyOpenSci’s current funding and structure
+
+I am grateful for 2 years of funding from the Sloan Foundation to 
+support this effort. Once I acquire additional funding, I 
+will be able to hire more staff to grow the organization. I am working with a 
+board of two right now: Tracy Teal (chair) and Karen Cranston. They both bring 
+tremendous experience building and supporting The Carpentries, among other 
+efforts; Tracy and Karen will guide me in propelling pyOpenSci forward. I also am 
+working with two incredible editors that have been crucial in supporting peer 
+review: David Nicholson and Ivan Ogasawara. In the upcoming months we will 
+recruit more editors to support our open source software review process. 
+Finally, I am also continuing to coordinate with and learn from the 
+well-established and successful rOpenSci. rOpenSci has set the precedent for 
+this type of organization in this space and I am grateful for their ongoing 
```

Comment:
"thises" in "this type of organization in this space" sound a bit repetitive

"rOpenSci has set a precedent for community driven review of software for open science"?